Common/Serialize/SerializeMap.h

```diff
@@ -152,9 +152,9 @@ void Do(PointerWrap &p, std::multimap<K, T> &x) {
 template<class K, class T>
 void Do(PointerWrap &p, std::unordered_multimap<K, T *> &x) {
 	if (p.mode == PointerWrap::MODE_READ) {
-		for (auto it = x.begin(), end = x.end(); it != end; ++it) {
-			if (it->second != nullptr)
-				delete it->second;
+		for (const auto &[_, value] : x) {
+			if (value != nullptr)
+				delete value;
```

Comment:
nit: Personally, I like to always pretend that `delete` mutates a pointer, and avoid `const` for things I'm going to delete.  Semantically, I think it'd be confusing to see a `const int *p` argument to a function which ends up actually deleting that pointer.

Anyway, it sometimes does in fact mutate the object by calling the destructor.

-[Unknown]

Reply:
If I'm not mistaken, delete works fine with the nullptr so the check is superfluous.